modules/viewing-scheduled-pipeline-runs.adoc

```diff
@@ -19,17 +19,16 @@ endif::[]
 * You have scheduled a pipeline run that is available to view.
 
 .Procedure
-. From the {productname-short} dashboard, click *Data Science Pipelines* -> *Runs*.
-+
-The *Runs* page opens.
-. From the *Project* list, select the project whose scheduled pipeline runs you want to view.
-. Click the *Schedules* tab.
+. From the {productname-short} dashboard, click *Experiments* -> *Experiments and runs*.
+. Select the *project* that contains the pipeline experiment for the scheduled pipeline runs that you want to view.
+. From the list of pipeline experiments, click the pipeline experiment that contains the scheduled pipeline runs that you want to view.
```

Comment:
Can omit the second 'pipeline' here.

Reply:
Omitted the second occurrence of 'pipeline'.